container/images/ruby/Dockerfile

```diff
@@ -0,0 +1,63 @@
+ARG BASE_IMAGE
+FROM ${BASE_IMAGE}
+
+ARG RUBY_VERSION
+ENV RUBY_VERSION="2.2.2"
+
+RUN /bin/bash -l -c "curl -sL https://deb.nodesource.com/setup_8.x | bash -"
+RUN /bin/bash -l -c "curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add -"
+RUN /bin/bash -l -c 'echo "deb https://dl.yarnpkg.com/debian/ stable main" | tee /etc/apt/sources.list.d/yarn.list'
+
+RUN apt-get update && apt-get install -qy \
+    autoconf \
+    automake \
+    bison \
+    build-essential \
+    curl \
+    g++ \
+    gawk \
+    gcc \
+    git-core \
+    libc6-dev \
+    libcurl4-openssl-dev \
+    libffi-dev \
+    libgdbm-dev \
+    libncurses5-dev \
+    libreadline-dev \
+    libreadline6-dev \
+    libsqlite3-dev \
+    libssl-dev \
+    libssl1.0.0 \
+    libtool \
+    libxslt1-dev \
+    libyaml-dev \
+    libxml2-dev \
+    make \
+    nodejs \
+    openssl \
+    patch \
+    pkg-config \
+    software-properties-common \
+    python-software-properties \
+    sqlite3 \
+    yarn  \
+    zlib1g-dev \
+    --no-install-recommends && rm -r /var/lib/apt/lists/*
+
+RUN /bin/bash -l -c "echo progress-bar >> ~/.curlrc"
+RUN curl -sSL https://rvm.io/mpapis.asc | gpg --import -
+RUN echo "Downloading and installing 'rvm'" && curl -sSL https://get.rvm.io | bash -s stable
```

Comment:
Why do we need both RVM and Rbenv?

Reply:
We need version 2.2.2 as per the project config and I didnt get any success using any other means, rvm and rbenv worked.

Maybe we don't need the patch installation on top of 2.2.2 - on that note is it possible to move to another version which can be installed more easily?